pkg/platform/runtime/buildscript/buildscript_test.go

```diff
@@ -287,10 +288,11 @@ main = runtime
     "let": {
       "runtime": {
         "solve": {
+          "at_time": "$at_time",
           "requirements": [
             {
               "name": "python",
-			  "namespace": "language"
+              "namespace": "language"
```

Comment:
I know this isn't related to your PR but shouldn't this be updated to the new `Req` format?

Reply:
No, this is the result of the translation from buildscript to JSON. The new Req format was on line 276.